### PR TITLE
Rc/generic inbound api basic log report

### PR DESCRIPTION
**Pull request by esoergel**
@Robert-Costello the alternate merge base thing I suggested got a little screwy, but this should fix it.

Merge resolution from https://github.com/dimagi/commcare-hq/pull/32297 and https://github.com/dimagi/commcare-hq/pull/32337

32337 had it's base branch as 32297, but 32297 was merged first, so when 32337 was merged, it was merged into this branch and not master.  This PR just merges that already reviewed branch into master.

### CODE REVIEW

**Comment by Robert-Costello** [comment]
> the alternate merge base thing I suggested got a little screwy, but this should fix it.

Thanks, Ethan. Things did get weird. I thought I had fixed it, but evidently not.

**Review by Robert-Costello — APPROVED** [review]

**Comment by Robert-Costello** [comment]
Went ahead and approved since the actual code changes are already approved by a third party. If it is better to wait for another approval that's fine.

**Comment by esoergel** [comment]
@Robert-Costello I think that's fine - this code has already been approved, as you said, which is also why I skipped the detailed PR description - it's just a mirror of https://github.com/dimagi/commcare-hq/pull/32337

**Pull request merged**
